pkg/driver/params.go

```diff
@@ -41,6 +44,13 @@ type VolumeParams struct {
 	PVName       string
 }
 
+// SnapshotParams holds collection of supported settings that can
+// be configured in snapshot class.
+type SnapshotParams struct {
+	SnapSize float64
+	Absolute bool
```

Comment:
let us name is AbsSnapSize to make it specify absolute for snapshot only. We might add few more parameters in future.